include/uv/errno.h

```diff
@@ -445,4 +445,10 @@
 # define UV__EILSEQ (-4027)
 #endif
 
+#if defined(ESTALE) && !defined(_WIN32)
+# define UV__ESTALE UV__ERR(ESTALE)
+#else
+# define UV__ESTALE (-10070)
```

Comment:
Why this number (not -4026)? Needs a rebase though anyways.

Reply:
I've mapped WSAESTALE system error code to that value.
Please see here
https://docs.microsoft.com/en-us/windows/win32/winsock/windows-sockets-error-codes-2#STALE_FILE_HANDLE_REFERENCE.

I don't mind to change value to 4026 if you think that it is the proper way of handling this here.

Reply:
Yes. Looks like error codes here is just a backward counter. Didn't notice that initially.